lib/mortar/list_command.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative "command"
+require_relative "mixins/client_helper"
+require_relative "mixins/tty_helper"
+
+module Mortar
+  class ListCommand < Mortar::Command
+    include Mortar::ClientHelper
+    include Mortar::TTYHelper
+
+    option ['-q', '--quiet'], :flag, "only output shot names"
+
+    def execute
+      shots = {}
```

Comment:
with `shots = Hash.new(0)` you can change the 

```ruby
        if shots[shot_name]
          shots[shot_name] += 1
        else
          shots[shot_name] = 1
        end
```
to just `shots[shot_name] += 1`

Reply:
what black magic is that?
```
2.4.2 :003 > {} == Hash.new(0)
 => true 
```